<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/images/lhlogo.png</normaloff>:/images/lhlogo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>610</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSub_system">
    <property name="title">
     <string>sub_system</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="actionTeacher"/>
    <addaction name="actionClass"/>
    <addaction name="actionStudent"/>
    <addaction name="actionCourse"/>
    <addaction name="actionUser"/>
   </widget>
   <addaction name="menuSub_system"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea|Qt::RightToolBarArea|Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionUser"/>
   <addaction name="actionTeacher"/>
   <addaction name="actionStudent"/>
   <addaction name="actionClass"/>
   <addaction name="actionCourse"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionTeacher">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/my.png</normaloff>:/icons/my.png</iconset>
   </property>
   <property name="text">
    <string>teacher</string>
   </property>
  </action>
  <action name="actionClass">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/class.png</normaloff>:/icons/class.png</iconset>
   </property>
   <property name="text">
    <string>class</string>
   </property>
  </action>
  <action name="actionStudent">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/learn.png</normaloff>:/icons/learn.png</iconset>
   </property>
   <property name="text">
    <string>student</string>
   </property>
  </action>
  <action name="actionCourse">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/task.png</normaloff>:/icons/task.png</iconset>
   </property>
   <property name="text">
    <string>course</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/portal.png</normaloff>:/icons/portal.png</iconset>
   </property>
   <property name="text">
    <string>login</string>
   </property>
  </action>
  <action name="actionUser">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/setting.png</normaloff>:/icons/setting.png</iconset>
   </property>
   <property name="text">
    <string>user</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
